internal/host/validator.go

```diff
@@ -534,6 +534,10 @@ func (v *validator) printBelongsToMajorityGroup(c *validationContext, status Val
 
 func (v *validator) IsNTPSynced(c *validationContext) ValidationStatus {
 
+	if swag.StringValue(c.cluster.HighAvailabilityMode) == models.ClusterHighAvailabilityModeNone {
```

Comment:
Maybe this should be a function? `ShouldCheckNTP`

Reply:
why to? it means for every place i will need this validation i will need to add function if should?

Reply:
It doesn't mean that, but actually having a generic dynamic switch-case wrapping all the validations, is required for profiling. :)